src/Service/HookService.php

```diff
@@ -88,11 +83,11 @@ public function getIntegrationModelFromContext()
         $cacheParams->setIdLang(Context::getContext()->language->id);
         $cacheParams->setIdShop(Context::getContext()->shop->id);
 
-        if ($this->projectCache->exist($cacheParams) && !$this->projectCache->isExpired($cacheParams)) {
-            $cacheResult = $this->projectCache->get($cacheParams);
-
-            return $cacheResult['content'];
-        }
+//        if ($this->projectCache->exist($cacheParams) && !$this->projectCache->isExpired($cacheParams)) {
```

Comment:
is it a mistake to comment this ?

Reply:
Yes, i tested without cache and i forgot to uncomment. Thank you a lot for review !